### PR TITLE
Issue 2676: Fix typos and remove .edit, .editonly annotations

### LINKED ISSUE #2676: Fix the use of {.edit} and {.edit-only} in the online tutorial pages

### PR DESCRIPTION
**Pull request by davidcok**
Fixes #2676

<small>By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT license](https://github.com/dafny-lang/dafny/blob/master/LICENSE.txt).</small>

### CODE REVIEW

**Review by atomb — CHANGES_REQUESTED** [review]
This looks great. I noticed one typo and have a couple of small questions.

File: docs/OnlineTutorial/Sequences.md [hunk 1]
```diff
@@ -90,22 +91,13 @@ method m()
...
-  assert s[|s|-1..|s|] == [5]; //slice just the last element, as a singleton
-  assert s[1..] == [2, 3, 4, 5]; // everything but the first
-  assert s[..|s|-1] == [1, 2, 3, 4]; // everything but the last
-  assert s == s[0..] == s[..|s|] == s[0..|s|] == s[..]; // the whole sequence
-```
-
 By far the most common operations on sequences are getting
-the first and last elements, and getting everything but the first and last
-elements, as these are often used in recursive functions, such as `sorted2`
+the first and last elements, and getting everything but the first cworand last
```
[the remark this answers is not in the document]
Comment (davidcok) [reply docs/OnlineTutorial/Sequences.md:95]: Aah - vi is often quicker for me, but I do get out of synch with the editing mode...

File: docs/OnlineTutorial/guide.md [hunk 2]
```diff
@@ -58,12 +59,12 @@ which takes a single integer parameter, called "`x`", and
...
 are required for each parameter and return value, and follow each name after a
 colon (`:`). Also, the return values are named, and there
-can be multiple return values, as in below:
+can be multiple return values, as in this code:
 
 ```dafny
 method MultipleReturns(x: int, y: int) returns (more: int, less: int)
 {
-  ...
+  ... ...
```
[the remark this answers is not in the document]
Comment (davidcok) [reply docs/OnlineTutorial/guide.md:67]: Should have been removed...

File: docs/OnlineTutorial/guide.md [hunk 3]
```diff
@@ -1478,15 +1217,15 @@ element type `T` (i.e., `array<T>`) and the null reference.
...
-Element access uses the standard bracket syntax and are indexed from
+Element access uses the standard bracket syntax and indexes from
 zero, so `a[3]` is preceded by the 3 elements `a[0]`,
 `a[1]`, and `a[2]`, in that order.
 All array accesses must be proven to be within bounds, which is part of Dafny's
 no-runtime-errors safety guarantee. Because bounds checks are proven
 at verification time, no runtime checks need to be made. To create a new array,
 it must be allocated with the `new` keyword, but for now
 we will only work with methods that take a previously allocated array as an
-argument. (See the tutorial on memory for more on allocation.)
```
[the remark this answers is not in the document]
Comment (davidcok) [reply docs/OnlineTutorial/guide.md:1489]: Correct.

**Review by atomb — APPROVED** [review]
Looks good!

**Pull request merged**